packages/core/content-type-builder/admin/src/components/FormModal/index.js

```diff
@@ -942,6 +942,13 @@ const FormModal = () => {
 
   const schemaKind = get(contentTypes, [targetUid, 'schema', 'kind']);
 
+  const isEditingFieldName =
```

Comment:
This calc will run on every render when it's not needed, why not always pass the event handler and run that check in said event handler?

Reply:
Ok, thanks for the suggestion, modified